Sources/Fluent/SQL/GeneralSQLSerializer.swift

```diff
@@ -402,7 +402,7 @@ open class GeneralSQLSerializer: SQLSerializer {
     }
 
 
-    open func sql(_ type: Schema.Field.DataType) -> String {
+    open func sql(_ type: Schema.Field.DataType, primary: Bool) -> String {
```

Comment:
I think we generally have been skirting swift conventions on bools, we should try to use `isPrimary` or even better `isPrimaryField` or `isPrimaryKey` or whatever this means.